packages/server/test/e2e/7_record_spec.js

```diff
@@ -406,6 +406,35 @@ describe('e2e record', () => {
     })
   })
 
+  context('empty specs', () => {
+    setupStubbedServer(createRoutes())
+
+    it('succeeds when empty spec file', async function () {
```

Comment:
you should link to the github issue that this is fixing, this is a specific regression